content/ko/docs/tasks/debug-application-cluster/logging-elasticsearch-kibana.md

```diff
@@ -0,0 +1,124 @@
+---
+reviewers:
+- piosz
+- x13n
```

Comment:
@pjhwa  - unless you know that these two reviewers speak Korean well, you should remove them from the Korean version of this page.

Reply:
Oops! I just forgot to remove that. I will. Thank you!